src/modules/pos/validator/components/SentStakes/SentStakes.js

```diff
@@ -60,10 +60,10 @@ const SentStakes = ({ history }) => {
         </Heading>
       </BoxHeader>
       <BoxContent>
-        <QueryTable
+        <Table
```

Comment:
whats the reason to change query table to the table ?

Reply:
At max one can have 10 stakes, not beyond that, so we just need table for this scenario.

Reply:
I guess it is the loadmore button you want to fix with this? 
The root cause of that is that `useSentStakes` uses `useCustomInfiniteQuery` instead of `useCustomQuery`, i have a refactored that in my branch.